src/test/java/com/gs/crdtools/SpecExtractorTest.java

```diff
@@ -0,0 +1,4 @@
+package com.gs.crdtools;
+
+public class SpecExtractorTest {
```

Comment:
Let's only check in code that actually does something

Reply:
This must've escaped my eyes while committing, apologies for that